modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3184,7 +3184,63 @@ Response::
 ==== Parameters
 No parameters.
 
+=== Show all delayed queries
 
+`GET /showdelayedlistall`
+
+When a query request is sent while another query is still running, it is sent to a "delayed" queue.
+Use this endpoint to get a list of all queries that have been requested but have not started running yet.
+
+Query timeout time is not affected by the time spent in the delayed queue.
+
+==== Sample request
+[tabs]
+====
+Request::
++
+[.wrap,console]
+----
+$ curl -X GET "http://localhost:9000/showdelayedlistall" |jq
+----
+Response::
++
+----
+{
+  "version": {
+    "edition": "enterprise",
+    "api": "v2",
+    "schema": 2
+  },
+  "error": false,
+  "message": "",
+  "results": [
+    {
+      "requestidPartial": "1670970545942",
+      "startTime": "2022-12-13 22:29:05.942",
+      "url": "/query/ldbc_snb/LTQ1?gap=60&qtf=vwlm_a_007",
+      "elapsedTime": 2023
+    },
+    {
+      "requestidPartial": "1670970545943",
+      "startTime": "2022-12-13 22:29:05.943",
+      "url": "/query/ldbc_snb/LTQ1?gap=60&qtf=vwlm_a_008",
+      "elapsedTime": 2022
+    },
+    {
+      "requestidPartial": "1670970545942",
+      "startTime": "2022-12-13 22:29:05.942",
+      "url": "/query/ldbc_snb/LTQ1?gap=60&qtf=vwlm_a_004",
+      "elapsedTime": 2023
+    },
+  ],
+  "code": "REST-0000"
+}
```

Comment:
Do we need consistency in the example responses.   Can we use the same graph/query examples as in /showprocesslistall documentation? https://docs.tigergraph.com/tigergraph-server/current/api/built-in-endpoints

Reply:
Can we change the query names to      
      "url": "/query/Work_Net/rngExample",
         "url": "/query/Entity_Resolution/rngExample",
     
     To be consistent across /showprocesslistall and /showdelayedlistall ?